Packs/VaronisDataSecurityPlatform/README.md

```diff
@@ -0,0 +1 @@
+TODO: Ask Shifra (techincal writer) to help us with read me file
```

Comment:
Please add this content prior to release.

Reply:
@mchasepan readme updated